addons/delivery/models/sale_order.py

```diff
@@ -173,6 +180,7 @@ def _action_confirm(self):
                 ('country_id', '=', country),
                 ('parent_id', '=', parent_id),
                 ('type', '=', 'delivery'),
+                ('is_pickup_location', '=', True),
```

Comment:
Mhm, I don't know about that.
Would stop working for all existing pickup point locations saved, and a new address would be created.

Reply:
Yes, but it is the future :)
Anyway, as it is a temporary solution, I can remove that.

Reply:
agree, I removed already :)